src/stores/messages/MessagesDataProviderStore.ts

```diff
@@ -137,6 +137,12 @@ export default class MessagesDataProviderStore {
 				}
 			}
 
+			const { selectedMessageId, scrollToMessage } = this.messagesStore;
+			this.searchChannelNext.onStop = () =>
```

Comment:
this code should be deleted

Reply:
when you're picking some event, message list is not scrolling to attached message without this part

Reply:
![2021-12-06_15-02](https://user-images.githubusercontent.com/74649472/144842776-b141e443-a360-42c9-a38b-ce3394e367ae.png)
for example

Reply:
then we should develop new appropriate solution to scroll to selected message